MonMaraicher/Features/FarmerDetail/FarmerDetailsView.swift

```diff
@@ -36,20 +37,22 @@ struct FarmerDetailsView: View {
                         .clipShape(.rect(cornerRadius: 16))
                         .shadow(radius: 2)
                         .allowsHitTesting(false)
+                    addressSection
+                        .background(.thinMaterial)
+                        .clipShape(.rect(cornerRadius: 20))
```

Comment:
Same here? Why not adding that in address section instead?

Reply:
It's true